SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Govdirectory Security Policy
+
+## Scope
+
+This policy is intended for all resources available on `*.govdirectory.org*` and any repository in the Govdirectory GitHub organization.
+
+## Reporting a Vulnerability
+
+You can privately report a vulnerability to the Govdirectory team by [creating a security advisory on GitHub](https://github.com/govdirectory/website/security). This report will be kept private while it is being assessed by the team.
```

Comment:
I think it is more helpful to give the direct link as GitHub is kind of convoluted.
```suggestion
You can privately report a vulnerability to the Govdirectory team by [creating a security advisory on GitHub](https://github.com/govdirectory/website/security/advisories/new). This report will be kept private while it is being assessed by the team.
```